test/integration/document/PresentationViewer.e2e.test.js

```diff
@@ -25,4 +27,17 @@ describe('Presentation Viewer', () => {
 
     });
 
+    it('Should navigate to a specified page when an internal link is clicked', () => {
+        cy.showPreview(token, fileWithLinksId);
+
+        // Assert document content is present and clickable
+        cy.contains('LINK TO PAGE 3');
```

Comment:
I believe you may be able to do `cy.contains('LINK TO PAGE 3').click();` as a single statement.

Reply:
For links to work in preview with PPTs, you have to add a shape, so the text isn't actually clickable.